src/components/configuration/BlueOSAlertsConfig.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <ExpansiblePanel :is-expanded="true" no-top-divider>
+    <template #title>BlueOS monitoring alerts:</template>
+    <template #info>
+      Configure alerts for BlueOS system monitoring including CPU temperature and usage thresholds. <br />
+      Set custom thresholds and minimum intervals between alerts to prevent spam.
+    </template>
+    <template #content>
+      <div class="ml-4 mb-4 mr-4">
+        <table class="w-full">
+          <thead>
+            <tr>
+              <th class="text-center text-sm font-medium text-slate-200 pb-3 w-12"></th>
+              <th class="text-center text-sm font-medium text-slate-200 pb-3 w-40">Alert Type</th>
+              <th class="text-center text-sm font-medium text-slate-200 pb-3 w-32">Threshold</th>
+              <th class="text-center text-sm font-medium text-slate-200 pb-3 w-32">Min Interval (s)</th>
+              <th class="text-center text-sm font-medium text-slate-200 pb-3 w-24">Level</th>
+            </tr>
+          </thead>
+          <tbody>
+            <!-- CPU Temperature Configuration -->
+            <tr class="border-b border-slate-600">
+              <td class="py-3 text-center">
+                <input
+                  v-model="vehicleAlerterStore.blueOsAlertsConfig['cpu-temperature'].enabled"
+                  type="checkbox"
+                  class="w-4 h-4"
+                />
+              </td>
+              <td class="py-3 text-center">
+                <span class="text-sm font-medium">CPU Temperature</span>
+              </td>
+              <td class="py-3 text-center">
+                <div class="flex items-center justify-center gap-2">
+                  <input
+                    v-model.number="vehicleAlerterStore.blueOsAlertsConfig['cpu-temperature'].threshold"
```

Comment:
I'm not 100% sure if this is the correct approach to do that, the system-information page has the /platform endpoint where it returns platform specific data, in this case, for the raspberry, it returns the events such as thermal throttle, under voltage, and others.
For a complete list, check here:
https://github.com/patrickelectric/linux2rest/blob/master/src/features/platform/raspberry.rs#L10

Reply:
Nice! I like the idea of implementing the undervoltage and throttling warns as well.

About allowing customization of the threshold for those that have continuous values, it is needed since each platform behaves differently, and the user will know better than anyone what is a safe level for their mission and when they need to be warned to take action.